backend/src/test/java/backend/mulkkam/intake/service/IntakeAmountServiceIntegrationTest.java

```diff
@@ -96,6 +103,41 @@ void error_memberIsNotExisted() {
                     () -> intakeAmountService.modifyTarget(intakeTargetAmountModifyRequest, Long.MAX_VALUE));
             assertThat(exception.getErrorCode()).isEqualTo(NotFoundErrorCode.NOT_FOUND_MEMBER);
         }
+
+        @DisplayName("해당 수정이 추천에 의한 수정일 경우 금일 목표에만 반영된다")
+        @Test
+        void success_recommendAmount() {
+            // given
+            int memberTargetAmount = 1500;
+            Member member = MemberFixtureBuilder
+                    .builder()
+                    .targetAmount(new Amount(memberTargetAmount))
+                    .build();
+            memberRepository.save(member);
+            IntakeHistory intakeHistory = IntakeHistoryFixtureBuilder
+                    .withMember(member)
+                    .date(LocalDate.now())
+                    .build();
+            intakeHistoryRepository.save(intakeHistory);
+            ModifyIntakeTargetAmountByRecommendRequest modifyIntakeTargetAmountByRecommendRequest = new ModifyIntakeTargetAmountByRecommendRequest
+                    (
+                            1000
+                    );
```

Comment:
```suggestion
            ModifyIntakeTargetAmountByRecommendRequest modifyIntakeTargetAmountByRecommendRequest = new ModifyIntakeTargetAmountByRecommendRequest(1000);
```
혹시 개행에 체체의 의도가 있었는 지 궁금해요!